sb/robot/sim_robot.py

```diff
@@ -156,15 +317,32 @@ def _apply_wheel_force(self, y_position, power):
                              force_magnitude * sin(self.heading))
         self._body.apply_force(force_world_space, location_world_space)
 
+    def _apply_wheel_coast(self, y_position):
```

Comment:
Is this just a special case of `_apply_wheel_force` with `power = 0` and a difference coefficient of friction? If so, can we merge the two functions instead of copying them?

Reply:
Fixed in https://github.com/sourcebots/simulator/pull/1/commits/d0f61ebb348445aad93eeffe38638080339bbdd6